### PR TITLE
RELATED: BB-509 TT footer

### PR DESCRIPTION
**Pull request by petrjanu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/360/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/215/ : SUCCESS in 1m 45s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/47/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/361/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/201/ : SUCCESS in 5m 12s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/362/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/216/ : UNSTABLE in 5m 23s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/379/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/232/ : FAILURE in 1m 39s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by BugsBunny338** [comment]
LGTM, I like the simplification of `shouldShowTotals`.

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/381/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/207/ : FAILURE in 11m 26s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/54/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/382/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/208/ : UNSTABLE in 4m 15s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/383/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/234/ : SUCCESS in 1m 41s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/384/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/235/ : SUCCESS in 2m 56s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/55/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/385/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/209/ : SUCCESS in 5m 18s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/386/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/236/ : SUCCESS in 3m 08s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/387/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/237/ : SUCCESS in 1m 37s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/56/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/388/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/210/ : SUCCESS in 4m 06s

File: src/Table/test/TableVisualization.spec.jsx [hunk 1]
```diff
@@ -182,13 +182,32 @@ describe('Table', () => {
         };
 
         describe('totals edit not allowed', () => {
+            it('should not has footer when no totals provided', () => {
```
Comment (martinmosko) [inline src/Table/test/TableVisualization.spec.jsx]: should not **display** footer when ... ?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/390/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/239/ : SUCCESS in 1m 37s

File: src/Table/test/TableVisualization.spec.jsx (hunk 1 above)
Comment (petrjanu) [reply src/Table/test/TableVisualization.spec.jsx]: It is checking only hasFooterWithTotals "flag", not rendering of footer. This "flag" is used for more features. Checking rendering of footer is pretty complex in this component and is covered by othe test cases. Thats why I used "has" :)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/391/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/240/ : SUCCESS in 1m 38s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/392/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/241/ : SUCCESS in 1m 41s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/394/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/243/ : SUCCESS in 1m 41s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/57/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/395/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/211/ : SUCCESS in 4m 06s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/397/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/245/ : SUCCESS in 1m 35s

**Comment by petrjanu** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-indigo-visualizations-storybook-zuul-docker/58/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-opensource-repository-check/398/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-pull-request-dispatcher/212/ : SUCCESS in 4m 03s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-indigo-visualizations-merge-compliance-check/93/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-indigo-visualizations-unit-tests-zuul-docker-el7/246/ : SUCCESS in 3m 21s

**Pull request merged**